### DISCUSSION TITLE
open and play setting?

### DISCUSSION THREAD

**Discussion by paxcoder** [losnoco/Cog#323] [Ideas]
Hi.
I want to get something like that, you just open a file, and it just automaticaly plays, ofcourse it would be configurable.

btw I updated the Polish translation, Lokalise is not that good with VoiceOver, so I am doing it really slowly.

**Comment by kode54** [comment]
It already does that by default. Only it doesn't take over file associations by default, because nothing can do that unless it's the first app for a given format.
